packages/components/package.json

```diff
@@ -36,8 +36,8 @@
     "@chakra-ui/react": "^2.6.1",
     "@chakra-ui/react-utils": "^2.0.11",
     "@chakra-ui/theme-tools": "^2.0.16",
-    "@emotion/react": "^11.11.1",
-    "@emotion/styled": "^11.11.0",
+    "@emotion/react": "11.6.0",
+    "@emotion/styled": "11.6.0",
```

Comment:
[Downgrading @emotion to v11.6.0 fixes test error](https://github.com/emotion-js/emotion/issues/2702#issuecomment-1084556410)

Reply:
[Fresh dependencies install has fixed it](https://github.com/cmpsr/composer/actions/runs/5393704403/jobs/9793814830?pr=746) :tada:

So I have rolled @emotion back to the latest.

[More on this comment](https://github.com/emotion-js/emotion/issues/2702#issuecomment-1303338890)